packages/wonder-blocks-dropdown/components/action-menu.js

```diff
@@ -191,41 +100,34 @@ export default class ActionMenu extends React.Component<MenuProps, State> {
             children,
             onChange,
             selectedValues,
+            style,
             "aria-disabled": ariaDisabled, // WB-535 avoids passing this prop to the opener
             /* eslint-enable no-unused-vars */
             ...sharedProps
         } = this.props;
-        const {open} = this.state;
-
-        const items = this.getMenuItems();
-
-        const opener = (
-            <ActionMenuOpener
-                {...sharedProps}
-                disabled={items.length === 0 || disabled}
-                onOpenChanged={this.handleOpenChanged}
-                open={open}
-                ref={this.handleOpenerRef}
-                testId={testId}
-            >
-                {menuText}
-            </ActionMenuOpener>
-        );
 
+        const menuItems = React.Children.toArray(this.props.children);
         return (
-            <DropdownCore
+            <Dropdown
                 alignment={alignment}
-                dropdownStyle={[styles.menuTopSpace, dropdownStyle]}
-                items={items}
-                keyboard={this.state.keyboard}
-                light={false}
-                onOpenChanged={this.handleOpenChanged}
-                open={open}
-                opener={opener}
-                openerElement={this.openerElement}
+                menuItems={menuItems}
+                onChange={this.props.onChange}
+                selectedValues={this.props.selectedValues}
```

Comment:
This could be `selectedValues={selectedValues}` since `selectedValues` is already being destructured above.